jupyterlab_caip_optimizer/src/components/parameter_details_dialog.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+  DialogActions,
+  Button,
+} from '@material-ui/core';
+import { ParameterSpec } from '../types';
+import { ParameterDetails } from './parameter_details';
+
+interface Props {
+  spec?: ParameterSpec;
+  onClose?: () => void;
+}
+
+const ParameterDetailsDialog: React.FC<Props> = ({ spec, onClose }) => {
+  return (
+    <Dialog
+      open={!!spec}
+      onClose={onClose}
+      data-testid="parameterDetailsDialog"
+    >
+      {!!spec && (
+        <>
+          <DialogTitle>Parameter "{spec.parameter}"</DialogTitle>
+          <DialogContent>
+            <DialogContentText>
+              The study config parameter specification.
+            </DialogContentText>
+            <ParameterDetails spec={spec} />
+          </DialogContent>
+          <DialogActions>
+            <Button onClick={onClose} color="primary" variant="contained">
```

Comment:
I think Exit / Cancel buttons are generally not primary in color (only action buttons that you want to direct attention to and will do stuff are raised/colored)

Reply:
Should I then remove the contained variant?